misc/scripts/remove.sh

```diff
@@ -56,7 +52,8 @@ case "$url" in
 	;;
 
 	*)
-		cd "$STOWDIR" || (sudo mkdir -p "$STOWDIR"; cd "$STOWDIR")
+		sudo mkdir -p "$STOWDIR"
+		cd "$STOWDIR" 2> /dev/null || ( fancy_message error "Could not enter ${STOWDIR}" && exit 1 )
```

Comment:
Add error logging

Reply:
okay, I'll add it in soon.

Reply:
Fixed in https://github.com/pacstall/pacstall/pull/276/commits/99e0e97dd340f72b3f73673958e5ead2e5370b67